iostat/iostat.go

```diff
@@ -180,13 +182,17 @@ func (iostat *IOSTAT) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
 
 // Init initializes iostat plugin
 func (iostat *IOSTAT) run(mts []plugin.MetricType) ([]string, map[string]float64, error) {
-	// TODO: reminder - remove these todo statements from the README (roadmap section) once
-	//       they are completed
-	// TODO: add validation that we are running sysstat version 10.2.0 or greater else print a
-	//       a warning or otherwise communicate to the user that they are potentially using an
-	//       unsupported/tested configuration
 	// TODO: allow the path and/or name of the command to be overriden through the pluginConfigType
 
+	versionString := iostat.cmd.Exec("iostat", []string{"-V"})
+	version, err := iostat.parser.ParseVersion(versionString)
+	if err != nil {
+		return nil, nil, err
+	}
+	if version[0] < 10 || (version[0] == 10 && version[1] < 2) {
```

Comment:
Consider checking of `version` length to make sure that it has sufficient number of elements